mountpoint-s3/src/inode.rs

```diff
@@ -1214,6 +1214,8 @@ pub struct InodeStat {
     pub atime: OffsetDateTime,
     /// Etag for the file (object)
     pub etag: Option<String>,
+    /// Storage class for the file (object), if known
+    pub storage_class: Option<String>,
```

Comment:
I worry the extra string here could be quite wasteful (and redundant!). Maybe we could just store the resulting permissions here instead of the storage class.

Reply:
I didn't want to inline the permissions here (our inode stats otherwise don't know about POSIX permissions). But this was a great chance to use my new favorite-named crate, [lasso](https://crates.io/crates/lasso), to intern the storage class names!

Reply:
Cool. Is that a Texan thing?